frontend/src/app/assess/[projectId]/[domain]/[practice]/page.tsx

```
@@ -204,22 +204,22 @@ export default function AssessmentPage() {
+                    <span className="inline-flex items-center px-3 py-1 rounded-full text-xs font-medium bg-muted text-muted-foreground">
                       Stream {question.stream}
                     </span>
                   </div>
-                  <p className="text-white font-medium text-lg leading-relaxed">
+                  <p className="text-card-foreground font-medium text-lg leading-relaxed">
                     {question.question}
                   </p>
                   {question.description && (
-                    <div className="mt-3 rounded-xl border border-dashed border-white/20 bg-white/10 p-4 text-sm text-gray-200 [&_h1]:text-lg [&_h1]:font-bold [&_h1]:mb-2 [&_h2]:text-base [&_h2]:font-bold [&_h2]:mb-2 [&_h3]:text-sm [&_h3]:font-bold [&_h3]:mb-1 [&_ul]:list-disc [&_ul]:ml-6 [&_ol]:list-decimal [&_ol]:ml-6 [&_li]:mb-1 [&_strong]:font-bold [&_em]:italic [&_u]:underline [&_a]:text-purple-300 [&_a]:underline [&_p]:mb-2 [&_p:last-child]:mb-0">
+                    <div className="mt-3 rounded-xl border border-dashed border-border bg-muted/50 p-4 text-sm text-muted-foreground [&_h1]:text-lg [&_h1]:font-bold [&_h1]:mb-2 [&_h2]:text-base [&_h2]:font-bold [&_h2]:mb-2 [&_h3]:text-sm [&_h3]:font-bold [&_h3]:mb-1 [&_ul]:list-disc [&_ul]:ml-6 [&_ol]:list-decimal [&_ol]:ml-6 [&_li]:mb-1 [&_strong]:font-bold [&_em]:italic [&_u]:underline [&_a]:text-primary [&_a]:underline [&_p]:mb-2 [&_p:last-child]:mb-0">
                       <div dangerouslySetInnerHTML={{ __html: safeRenderHTML(question.description) }} />
```

Biome (2.1.2)

[error] Avoid passing content using the dangerouslySetInnerHTML prop.

Setting content using code can expose users to cross-site scripting (XSS) attacks

(lint/security/noDangerouslySetInnerHtml)